src/export/bops/tests/userRole.test.ts

```diff
@@ -11,13 +11,7 @@ const mockFlow = {
       fn: "user.role",
       text: "user role",
     },
-    edges: [
-      "applicant_id",
-      "agent_id",
-      "proxy_id",
-      "unsupported_id",
-      "other_id",
-    ],
+    edges: ["applicant_id", "agent_id", "proxy_id", "unsupported_id"],
```

Comment:
Dropping `other_id` here was a requirement as `StaticSessionState` threw an error due to this being an invalid flow - pretty handy!